manifests/ubuntu.pp

```diff
@@ -46,6 +49,7 @@
     refreshonly => true,
     tries       => 2, # https://bugs.launchpad.net/launchpad/+bug/1430011 won't get fixed until 16.04 xenial
     try_sleep   => 30,
+    require     => File['/etc/apt/sources.list.d/datadog-beta.list'],
```

Comment:
Why do we require the beta list to update the agent ?

Reply:
Because the resource ensure's the file will be `absent` - the file resource is required, and then the file is `ensure => absent`.